examples/user_interaction/client/src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
What would you think about removing this service worker? It is a cool feature, but doesn't seem central to the demo of user interaction tracing